### DISCUSSION TITLE
When an error occurs in a loop

### DISCUSSION THREAD

**Discussion by PhD-GOAT** [neuropsychology/NeuroKit#911] [Q&A]
`Traceback (most recent call last):
  File ".py", line 48, in <module>
    df4 = nk.eog_analyze(eog1, sampling_rate=fs)
  File ".py", line 108, in eog_analyze
    features = eog_eventrelated(data)
  File "\neurokit2\eog\eog_eventrelated.py", line 73, in eog_eventrelated
    epochs = _eventrelated_sanitizeinput(epochs, what="eog", silent=silent)
  File "\neurokit2\epochs\eventrelated_utils.py", line 19, in _eventrelated_sanitizeinput
    raise ValueError(
ValueError: It seems that you are trying to pass a single epoch to an eventrelated function.If this is what you want, please wrap it in a dict: `{"0": epoch}`

Process finished with exit code 1`

I am running an algorithm on several files with your API. Some files display messages like that. Is there something in your API that allows you to move to another file when an error occurs? Or do I have to implement it in code?

**Comment by DominiqueMakowski** [comment]
Hi, I am not sure what you are trying to do (or what your question is). But this is the error:

```
 ValueError: It seems that you are trying to pass a single epoch to an eventrelated function.If this is what you want, please wrap it in a dict: {"0": epoch}
```

Please provide a reproducible code snippet
